### PR TITLE
TF-2135 Add splash screen for web

### PR DESCRIPTION
**Pull request by dab246**
## Issue 

#2135 

## Resolved

https://github.com/linagora/tmail-flutter/assets/80730648/b92f8670-3dbb-441d-9ff4-deaeff8091e8

### CODE REVIEW

**Review by drminh2807 — APPROVED** [review]

**Review by nqhhdev — APPROVED** [review]

File: assets/icons/icon_logo.png
Comment (hoangdat) [inline assets/icons/icon_logo.png:1]: it must be `Team Mail`

**Review by hoangdat — COMMENTED** [review]
IMO, the loading icon is big, can we reduce it

File: assets/icons/icon_logo.png
Comment (dab246) [reply assets/icons/icon_logo.png:1]: We need design for it

**Comment by dab246** [comment]
> IMO, the loading icon is big, can we reduce it

I reduce it. Please see demo again

**Comment by hoangdat** [comment]
Can we remove the linear loading at the top of the splash screen? Not the same in fluffy

**Comment by hoangdat** [comment]
> That is flutter's default in `debug` mode. @hoangdat It will not appear in `release` mode

forgot :))))

**Comment by hoangdat** [comment]
but how about the Team Mail :)))

**Comment by dab246** [comment]
> but how about the Team Mail :)))

We don't have a design for the logo with the text `Team Mail` below. Please inform design so they can help.

**Comment by dab246** [comment]
> but how about the Team Mail :)))

Updated new design. Please review it.

**Pull request merged**